cli/build.clj

```diff
@@ -49,13 +52,74 @@
              :main 'clojure-lsp.main
              :basis basis})))
 
-(defn ^:private bin [opts]
+(defn ^:private l4j-xml
```

Comment:
Not sure it's a good idea to add all this complexity about launch4j, can't we make deps-bin work for windows as well? I'm the maintainer of that lib so issue and PRs are very welcome, WDYT?

Reply:
This will need some more time and effort to investigate whether it is possible `deps-bin` to support windows.

From a cursory look that I had, `deps-bin` appears to create a shell script that invokes the embedded jar code. Not sure how to translate this into a windows batch file.

The `lanuch4j` concept is not so complex btw, it just creates an xml file with the actual input, output and jre locations and invokes `launch4jc.exe` on it. 

Let me know how you want to proceed, trying to replicate what `deps-bin` on Windows looks like a task on its own which perhaps we can look separately?

Reply:
Would it help if babashka fs removed :const?

Reply:
We can proceed but i'd like to have this supported on deps-bin, LMK if you wanna help with that

Reply:
> We can proceed but i'd like to have this supported on deps-bin, LMK if you wanna help with that

Sure, I will try to have a look as part of #1211 and see how far we can go with it on windows, thanks!